### PR TITLE
update sdk

### PR DESCRIPTION
**Pull request by jiyuujin**
- update pedantic_mono
- add to flutter_lints
- lint by code

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -6,27 +6,27 @@ publish_to: 'none'
...
+  intl: ^0.17.0
+  transparent_image: ^2.0.0
 
 dev_dependencies:
+  build_runner:
+  flutter_gen_runner:
   flutter_test:
     sdk: flutter
-  pedantic_mono: ^1.11.0+2
+  pedantic_mono: ^1.14.0
```
Comment (zuct) [inline pubspec.yaml:28]: This is not what was meant by #39.

Comment (jiyuujin) [reply pubspec.yaml:28]: @zuct [pedantic_mono](https://pub.dev/packages/pedantic_mono) is added a rule for flutter_lints.

Comment (jiyuujin) [reply pubspec.yaml]: @zuct use flutter_lints instead

**Review by granoeste — APPROVED** [review]

**Comment by jiyuujin** [comment]
マージします

**Pull request merged**